AK/SIMDExtras.h

```diff
@@ -101,6 +101,38 @@ ALWAYS_INLINE static int maskcount(i32x4 mask)
 
 // Load / Store
 
+template<SIMDVector VectorType>
+ALWAYS_INLINE static VectorType load_unaligned(void const* a)
+{
+    VectorType v;
+    __builtin_memcpy(&v, a, sizeof(VectorType));
+    return v;
+}
+
+template<SIMDVector VectorType>
+ALWAYS_INLINE static void store_unaligned(void* a, VectorType const& v)
+{
+    // FIXME: Does this generate the right instructions?
+    __builtin_memcpy(a, &v, sizeof(VectorType));
+}
+
+template<SIMDVector TDst, SIMDVector TSrc>
+ALWAYS_INLINE static TDst as(TSrc const& v)
```

Comment:
This is essentially a `bit_cast` please prefer that

Reply:
OK, reverted, used `bit_cast` instead.